modules/EnsEMBL/Web/Component/Help/Biotypes.pm

```diff
@@ -0,0 +1,73 @@
+=head1 LICENSE
+
+Copyright [1999-2015] Wellcome Trust Sanger Institute and the EMBL-European Bioinformatics Institute
+Copyright [2016-2018] EMBL-European Bioinformatics Institute
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+     http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+
+=cut
+
+package EnsEMBL::Web::Component::Help::Biotypes;
+
+use strict;
+use warnings;
+no warnings "uninitialized";
+
+use base qw(EnsEMBL::Web::Component::Help);
+
+sub _init {
+#  my $self = shift;
+#  $self->cacheable( 0 );
+#  $self->ajaxable(  0 );
+#  $self->configurable( 0 );
```

Comment:
Not sure if any of this is relevant or whether this can be safely removed.

Reply:
I don't know that all of it is relevant, but it won't break anything if you uncomment it. I think, though, that you might want to set 'cacheable' to 1 since the page isn't going to change much between releases.

Reply:
Also, just as a coding style tip, you don't need the curly braces around interpolated scalar variables, eg. in lines 41 and 56 below. However they are recommended when dereferencing variables, mainly so that you can write e.g. @{$children||[]} (line 52) - because Perl is untyped, you can't rely on $children to be an array reference (even if you wrote the code that creates $children, because someone else could modify it later and screw things up!). And since the most likely "wrong" return value is undef, this "or []" construction prevents ugly code barfs.